cdap-ui/app/cdap/components/InfoCard/index.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {PropTypes} from 'react';
+require('./InfoCard.less');
+var classNames = require('classnames');
+
+const propTypes = {
+ primaryText: PropTypes.string,
+ secondaryText: PropTypes.string,
+ superscriptText: PropTypes.string,
+ isLoading: PropTypes.bool
+};
+
+function InfoCard({isLoading, primaryText, secondaryText, superscriptText}) {
+
+  var textClasses = classNames("info-card-text", {'hidden' : isLoading});
+  var superScriptClasses = classNames("superscript-text", {'hidden' : isLoading});
+  var superscript = <div className={superScriptClasses}> {superscriptText} </div>;
```

Comment:
Any specific reason this is in a separate variable rather than inside the return jsx?